packages/patterns/src/patterns/patternMatchers.js

```diff
@@ -1886,7 +1917,7 @@ const InterfaceGuardPayloadShape = M.splitRecord(
   {
     interfaceName: M.string(),
     methodGuards: M.recordOf(M.string(), MethodGuardShape),
-    sloppy: M.boolean(),
+    defaultGuards: M.or('never', 'passable', 'raw'),
```

Comment:
Since this property is new, consider whether it should be moved into the optional part of this `splitRecord` pattern. But if you are confident we don't need to, then don't.

Reply:
Thanks, I reintroduced sloppy and moved defaultGuards down to remain compatible.

Reply:
Actually, you introduced `defaultGuards` in the mandatory section. If we get away with that, then we know this new code never encounters any `InterfaceGuardPayload`s created by code before this PR. That's great! If we believe that, then I think you can simply delete the optional `sloppy:` below. Otherwise, I think both `sloppy:` and `defaultGuards:` need to go into the optional section.